test/statsCases/performance-error/index.js

```diff
@@ -0,0 +1,4 @@
+require("./a");
+require(["./b"]);
+require(["./c"]);
+require("lodash");
```

Comment:
`lodash`? Where is that coming from? 😁 

You could also create a dummy file that exceeds 250 kb and delete it afterwards...